gitpod-io/website
# Remove references to DataGrip as it's not planned to be supported by JetBrains Gateway

## Description

Remove references to DataGrip as it's not planned to be supported by JetBrains Gateway.

We've confirmed it with JetBrains team.

## Related Issue(s)

None.

## How to test

Open the preview environment of the Website and confirm if there's no more reference to DataGrip.

## Release Notes

```release-note
NONE
```

## Documentation
